app/controllers/concerns/paginable.rb

```diff
@@ -38,7 +38,7 @@ def paginable_base_url(page = 1)
     # Generates an HTML link to sort given a sort field.
     # sort_field {String} - Represents the column name for a table
     def paginable_sort_link(sort_field)
-      return link_to(sort_link_name(sort_field), sort_link_url(sort_field), 'data-remote': true)
+      return link_to(sort_link_name(sort_field), sort_link_url(sort_field).html_safe, 'data-remote': true)
```

Comment:
sort_link_name is already safe html through raw which is indeed a wrapper for html_safe method, therefore is not needed to add .html_safe here

Reply:
for some reason it was coming through in the tests as `sort_direction=asc&amp;sort_field=email`. This fixed that issue and didn't seem to have any adverse affect on the html page. Perhaps the way the test is accessing the results outside of the normal UI workflow is the issue. The test is imitating the ajax call but without the original context of the page with its default sort rules.